featuretools/tests/primitive_tests/test_feature_serialization.py

```diff
@@ -17,8 +17,8 @@
 BUCKET_NAME = "test-bucket"
 WRITE_KEY_NAME = "test-key"
 TEST_S3_URL = "s3://{}/{}".format(BUCKET_NAME, WRITE_KEY_NAME)
-S3_URL = "s3://featuretools-static/test_feature_serialization_1.0.0"
```

Comment:
What happened when trying to load features that used an older entityset schema?

Reply:
It fails when it tries to deserialize the entityset from the features dictionary